src/dialects/oracle/schema/columncompiler.js

```diff
@@ -26,14 +26,14 @@ assign(ColumnCompiler_Oracle.prototype, {
     });
   },
 
-  increments () {
+  increments ({ primaryKey = true } = {}) {
     this._createAutoIncrementTriggerAndSequence();
-    return 'integer not null primary key';
+    return primaryKey ? 'integer not null primary key' : 'integer not null';
```

Comment:
this most probably doesnt create auto incrementing column but just integer that is not null